packages/features/ee/teams/components/MemberList.tsx

```diff
@@ -377,6 +377,11 @@ export default function MemberList(props: Props) {
           return filterValue.includes(rows.getValue(id));
         },
       },
+      {
+        id: "lastActiveAt",
+        header: "Last Active",
```

Comment:
eventually we have to translate these but i see all headers are hard coded english right now

Reply:
I think a separate PR so translate all headers is the way to go